docs/introduction.rst

```diff
@@ -0,0 +1,88 @@
+************
+Introduction
+************
+
+Overview
+========
+
+``Remme`` is a blockchain-based protocol used for issuing and management of X.509 client certificates to resolve issues
```

Comment:
Not X.509 certs, public keys in general.

Reply:
@eugene-babichenko, done.